client/src/components/Attachment/AttachmentCard.js

```diff
@@ -0,0 +1,121 @@
+import { gql } from "@apollo/client"
+import { Icon } from "@blueprintjs/core"
+import { IconNames } from "@blueprintjs/icons"
+import API from "api"
+import ConfirmDestructive from "components/ConfirmDestructive"
+import LinkTo from "components/LinkTo"
+import PropTypes from "prop-types"
+import React from "react"
+import { Card } from "react-bootstrap"
+import { toast } from "react-toastify"
+import utils from "utils"
+import "./Attachment.css"
+
+const GQL_DELETE_ATTACHMENT = gql`
+  mutation ($uuid: String!) {
+    deleteAttachment(uuid: $uuid)
+  }
+`
+
+const AttachmentCard = ({
+  file,
+  edit,
+  remove,
+  setError,
+  setRemove,
+  uploadedList,
+  setUploadedList
+}) => {
+  const { backgroundSize, backgroundImage } = utils.getAttachmentIconDetails(
+    file,
+    true
+  )
+
+  return (
+    <div key={file.uuid} style={{ width: "20%" }}>
+      <Card>
+        <div
+          className="imagePreview info-show card-image"
+          style={{
+            backgroundSize,
+            backgroundImage: `url(${backgroundImage})`
+          }}
+        >
+          <div className="file-info image-info">
+            <div style={{ display: "grid" }}>
+              <LinkTo
+                className="detail-btn"
+                modelType="Attachment"
+                model={file}
+              >
+                {" "}
+              </LinkTo>
+            </div>
+          </div>
+        </div>
+        <Card.Body className="p-1 d-block">
+          <Card.Title style={{ fontSize: "15px" }} className="info-line">
+            {file?.fileName?.substring(0, 8)}...
+            <span>{utils.humanReadableFileSize(file?.contentLength)}</span>
+          </Card.Title>
+          {edit && (
+            <div className="info-line">
+              <div>
+                <LinkTo
+                  modelType="Attachment"
+                  edit
+                  model={file}
+                  button="outline-primary"
+                >
+                  <Icon icon={IconNames.EDIT} className="icon edit" />
+                </LinkTo>
+              </div>
+              <ConfirmDestructive
+                onConfirm={() => deleteAttachment(file.uuid)}
+                objectType="attachment"
+                objectDisplay={"#" + file.uuid}
+                title="Delete attachment"
+                variant="outline-danger"
+                buttonSize="xs"
+              >
+                <Icon icon={IconNames.TRASH} />
+              </ConfirmDestructive>
+            </div>
+          )}
+        </Card.Body>
+      </Card>
+    </div>
+  )
+
+  function deleteAttachment(uuid) {
+    const newAttachments = uploadedList.filter(item => item.uuid !== uuid)
+    API.mutation(GQL_DELETE_ATTACHMENT, { uuid })
+      .then(data => {
+        setUploadedList(newAttachments)
+        if (!remove) {
+          setRemove(true)
+        }
+        toast.error("Your attachment has been successfully deleted")
```

Comment:
Error toasts are for errors, not for success. Red pop-ups alert the user to something that's wrong, and should not be used to report back a successful result of a user action.